NuGet.config

```diff
@@ -3,7 +3,12 @@
   <packageSources>
     <add key="ASP.NET 5 Dev" value="https://www.myget.org/F/aspnetvnext/" />
     <add key="ASP.NET 5" value="https://www.myget.org/F/aspnetrelease/" />
-    <add key="nuget.org feed" value="https://www.nuget.org/api/v2/" />    
+    <add key="dotnet.myget.org dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/" />
+    <add key="AI myget" value="https://www.myget.org/F/applicationinsights/api/v3/index.json" />
+    <add key="dotnet.myget.org dotnet-cli" value="https://dotnet.myget.org/F/dotnet-cli/" />
+    <add key="coreclr-xunit" value="https://www.myget.org/F/coreclr-xunit/api/v2" />
+    <add key="AspNetCIDev" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
```

Comment:
We will apparently add nuget feed when we publish in nuget, and remove all the local feeds.